zzupy/app/eas/undergrad.py

```
@@ -0,0 +1,378 @@
+                logger.error("服务器返回消息 {}", response_data["msg"])
+                raise OperationError(f"服务器返回消息 {response_data['msg']}")
+
+        except httpx.HTTPStatusError as exc:
+            logger.error(
+                "{}请求返回失败状态码: {}",
+                self.CURRENT_SEMESTER_URL,
+                exc.response.status_code,
+            )
+            raise OperationError(
+                f"服务器返回错误状态 {exc.response.status_code}"
+            ) from exc
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)